Maple2.Server.DebugGame/Graphics/DebugFieldWindow.cs

```diff
@@ -170,11 +177,12 @@ public unsafe void OnRender(double delta) {
         #region Render code
         // Begin region for render code
 
-        Context.VertexShader!.Bind();
-        Context.PixelShader!.Bind();
-        Context.SampleTexture!.Bind();
-        Context.CoreModels!.Quad.Draw();
+        // Render 3D field content first (before ImGui)
+        if (ActiveRenderer is not null) {
+            ActiveRenderer.RenderFieldWindow3D(delta);
```

Comment:
This function and the contents in it could be merged with
https://github.com/AngeloTadeucci/Maple2/compare/master...mettaursp:Maple2:debug-drawing